libs/search-widget/src/viewer/previewers/Pdf.svelte

```diff
@@ -110,15 +110,13 @@
 
   const find = (query: string) => {
     // TODO: pdf search is still not reliable enough
```

Comment:
should we remove this TODO now the code is uncommented?

Reply:
In this case it's in purpose, pdf search still need more work.